src/docfx/build/xref/XrefResolver.cs

```diff
@@ -55,8 +59,12 @@ internal class XrefResolver
                                 buildScope,
                                 jsonSchemaTransformer.Value).Build());
 
+            var externalXrefMapAndExternalXref = ExternalXrefMapLoader.Load(config, fileResolver, errorLog);
```

Comment:
Lazy this `ExternalXrefMapLoader` call. Maybe you'll need a new type like `ExternalXrefMap` to abstract the result of `ExternalXrefMapLoader.Load`